File: Homework/Objective-C/HW7/HW7/Swimmer.m
//
//  Swimmer.m
//  HW7
//
//  Created by Руслан on 18.09.16.
//  Copyright © 2016 Ruslan Timchenko. All rights reserved.
//

#import "Swimmer.h"

@implementation Swimmer

- (NSString*) swim {
    return [NSString stringWithFormat:@"Swimmer %@ is swimming", self.name];
}

- (NSString*) laugh {
    return [NSString stringWithFormat:@"Swimmer %@ is laughing", self.name];
}

@end

File: Homework/Swift/HW7/HW7/Swimmer.swift
//
//  Swimmer.swift
//  HW7
//
//  Created by Руслан on 19.09.16.
//  Copyright © 2016 Ruslan Timchenko. All rights reserved.
//

import Foundation

class Swimmer: Swimmers {
    
    let name: String
    
    init(name: String) {
        self.name = name
    }
    
    func swim() -> String {
        return "Swimmer \(self.name) is swimming"
    }
}
